homeassistant/components/unifi/strings.json

```diff
@@ -33,6 +33,27 @@
       "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
     }
   },
+  "entity": {
+    "sensor": {
+      "device_state": {
+        "state": {
+          "disconnected": "Disconnected",
+          "connected": "Connected",
+          "pending": "Pending",
+          "firmware_mismatch": "Firmware Mismatch",
+          "upgrading": "Upgrading",
+          "provisioning": "Provisioning",
+          "heartbeat_missed": "Heartbeat Missed",
+          "adopting": "Adopting",
+          "deleting": "Deleting",
+          "inform_error": "Inform Error",
+          "adoption_failed": "Adoption Failed",
+          "isolated": "Isolated",
+          "unknown": "Unknown"
```

Comment:
unknown should not be its own state. Rather, the entity should return `None` in that case (which for the end user would still resolve to `unknown`, but the difference is, is that `unknown` already is a valid state and shown in the automation editor)